internal/pkg/helper/tokens.go

```diff
@@ -0,0 +1,93 @@
+package helper
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"github.com/assetto-corsa-web/accweb/internal/pkg/cfg"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+func GenerateTokenKeysIfNotPresent(config *cfg.Config) {
+	// Based on https://stackoverflow.com/questions/64104586/use-golang-to-get-rsa-key-the-same-way-openssl-genrsa
+	bitSize := 4096
+
+	logrus.Info("accweb: checking for secrets...")
```

Comment:
IMHO, in a helper package, make more sense handle exceptions with errors and just print logs in a higher level...  But I am ok with this implementation

Reply:
Yeah, `helper` might be the wrong location for the function, since it depends on the Config object as well. Do you have a better idea of where it should live?

Reply:
you can pass the path as parameters...

Reply:
`func GenerateTokenKeysIfNotPresent(privateKeyPath, publicKeyPath string) {`